hubverse-aws-upload/hubverse-aws-upload.yaml

```diff
@@ -57,15 +57,24 @@ jobs:
           'hub-config'
           'model-abstracts'
           'model-metadata'
+          # temporarily include model-output here
           'model-output'
           'target-data'
         )
         for DIRECTORY in "${hub_directories[@]}"
         do
-          if [ -d "./$DIRECTORY" ]; then aws s3 sync "./$DIRECTORY" "s3://$BUCKET_NAME/$DIRECTORY" --delete; fi
+          if [ -d "./$DIRECTORY" ]
+          then
+            rclone sync \
+              "./$DIRECTORY/" \
+              ":s3,provider=AWS,env_auth:$BUCKET_NAME/$DIRECTORY" \
+              --checksum --verbose --stats-one-line --config=/dev/null
+          fi
         done
+        # unlike other data, model-outputs are synced to a "raw" location
```

Comment:
Added an explicit step for model-outputs, so we can land those files in a separate location (because we want to do some transforms on them).

Reply:
Is it a default behavior for all the hubs?

Reply:
It is, with a plan to "fast follow" with the parquet conversion functionality that will put transformed data into `s3://hub-bucket-name/model-output`

Happy to rename it to something better...`raw` is a nomenclature commonly used in corporate data lakes, so that's what popped out of my head 😄 

I originally had the idea to copy the original files to both `s3://hub-bucket-name/raw/model-output` and `s3://hub-bucket-name/model-output` until the parquet transformation is in place, but then realized that would just be creating a bunch of data in the latter location that would require cleanup.

Reply:
I am ok with the name, I just wonder because some hubs might already be in a parquet format so it will not be necessary to copy it to `raw` and than transform it to `parquet`. It should be directly copied to `s3://hub-bucket-name/model-output`, no? (for example, the US SMH RSV Hub, is already in parquet format). 
So I wonder if it makes sense to have it as a default behavior.

Reply:
Ah I see--thanks for clarifying!

Nick had the same question, and my .02: we should always have a place like `raw` to store the model-outputs as they were originally submitted: https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-cloud/issues/20#issuecomment-1948768842